index.js

```diff
@@ -130,7 +130,10 @@ function asPromise(opts) {
 			}
 
 			if (err) {
-				err.response = response;
+				Object.defineProperty(err, 'response', {
+					value: response,
+					enumerable: false
```

Comment:
Sure. Up to you. It's clear for me since I know that all the configs are false by default.